hpx/runtime/runtime_fwd.hpp

```diff
@@ -15,15 +15,16 @@
 namespace hpx
 {
     class HPX_API_EXPORT runtime;
-
-    ///////////////////////////////////////////////////////////////////////////
-    class HPX_API_EXPORT runtime_impl;
+    class HPX_API_EXPORT runtime_distributed;
 
     /// The function \a get_runtime returns a reference to the (thread
     /// specific) runtime instance.
     HPX_API_EXPORT runtime& get_runtime();
     HPX_API_EXPORT runtime*& get_runtime_ptr();
 
+    HPX_API_EXPORT runtime_distributed& get_runtime_distributed();
+    HPX_API_EXPORT runtime_distributed*& get_runtime_distributed_ptr();
```

Comment:
Same here, do we always need this functions to be defined?

Reply:
No. I think this was a result of me planning to make this a runtime option, in which case the distributed runtime might not be available. As a compile time option only this doesn't make sense anymore.